Utils/Extensions/NSURLExtensions.swift

```diff
@@ -207,14 +207,16 @@ extension NSURL {
         return self
     }
 
-    public func extractDomainName() -> String {
-        let urlString =  self.normalizedHost() ?? self.URLString
-        var arr = urlString.componentsSeparatedByString(".")
-        if (arr.count >= 2) {
-            arr.popLast()
-            return arr.joinWithSeparator(".")
+    /**
+     * Returns the second level domain (SLD) of a url. It removes any subdomain/TLD
+     *
+     * E.g., https://m.foo.com/bar/baz?noo=abc#123  => foo
+     **/
+    public func hostSLD() -> String {
```

Comment:
Apparently SLD is a real thing, so that name works for me :)

Nit: how about a dynamic `var` rather than a `func`?